### PR TITLE
#165940869 Add number of articles per page

### PR DESCRIPTION
**Pull request by Ogutu-Brian**
### What does this PR do?
- Adds number of articles per page in paginated data
### What are the tasks to be completed?
* Ensure the endpoint for getting all articles has a count of the number of articles per page
### How can this be manually tested?
a. **After cloning the repo:-**
```
$ git checkout bg-add-articlescount-perpage-165940869
$ python manage.py runserver
```
b. **Test with postman**
1. After signing up a user and creating articles, using postman request:
```
GET http://127.0.0.1:8000/api/articles/
```
**Screenshots**
image
image

### What are the relevant pivotal tracker stories?
[#165940869](https://www.pivotaltracker.com/story/show/165940869)

### CODE REVIEW

**Review by matthenge — APPROVED** [review]
LGTM

**Review by mdn-network — CHANGES_REQUESTED** [review]
LGTM just fix the conflicts

**Comment by Ogutu-Brian** [comment]
> LGTM just fix the conflicts

Hello @Philipotieno ,
I just resolved them

**Review by mdn-network — APPROVED** [review]

**Review by IssaIan — APPROVED** [review]

**Review by Amoswachira — APPROVED** [review]

**Review by a-braham — APPROVED** [review]
LGTM

File: authors/apps/articles/serializers.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ class ArticlePaginator(PageNumberPagination):
 
     def get_paginated_response(self, data):
         return Response(OrderedDict([
+            ("pageArticlesCount", len(self.page.object_list)),
```
Comment (oluwagbenga-joloko) [inline authors/apps/articles/serializers.py]: pageArticlesCount is ambiguous. why not just pageCount or articlesPageCount?

Comment (Ogutu-Brian) [reply authors/apps/articles/serializers.py]: Let me fix that @oluwagbenga-joloko

**Pull request merged**